### ISSUE TITLE
Alt Text Localization

### ISSUE THREAD

**Issue by LucaArgentieri** [sanity-io/sanity-plugin-media#265]
**Is your feature request related to a problem? Please describe.**
Alt text should be translated for a multilingual website to maintain accessibility and SEO for each language, so it makes sense to create plugin fields for each language in the studio

**Describe the solution you'd like**
I found this plugin (sanity-plugin-media fork) which is already a good base to start with: https://github.com/DavithkbY/sanity-plugin-media-i18n

```js

const locales = [{name: 'English', id: 'en'}, {name: 'Dutch', id: 'nl'}]

export default defineConfig({
  plugins: [
    media({
      locales: locales,
    })
  ]
})
```

![Feature preview](https://private-user-images.githubusercontent.com/71630103/355802179-22ded7fe-4d9e-43e1-9561-8d9ff532441a.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTc5NjQ4NDcsIm5iZiI6MTc1Nzk2NDU0NywicGF0aCI6Ii83MTYzMDEwMy8zNTU4MDIxNzktMjJkZWQ3ZmUtNGQ5ZS00M2UxLTk1NjEtOGQ5ZmY1MzI0NDFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTE1VDE5MjkwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY1NDYwYjc4MzAwYWMwYzNmNjVmYmM4ZmY5OGFiNzNkZWY2YWNkY2JiNWQ1ZWI5MmNjZTE5ZWVkNWI4OThkM2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.-89aJ_EiW6YBnSfeFfWfhquUrlBFgIJkJuhUt_tZikE)

I think it's best to have another version of `DialogAssetEdit` if locales is inserted so you can work with locales and not create confusion
